### DISCUSSION TITLE
Is there a desktop version?

### DISCUSSION THREAD

**Discussion by TheCheddarCheese** [LiveSplit/LiveSplitOne#807] [Help]
I've seen people mention desktop in some issues, but I cant seem to find anything about that in the readme

**Comment by KebabLord** [comment]
the repo mentioned in issues is probably [this one](https://github.com/CryZe/livesplit-one-desktop).

**Reply by Ronin546** [reply]
There is already a desktop version.

**Reply by KebabLord** [reply]
When this github discussion was created and i gave my answer, the official desktop version didn't exist yet Einstein.
